### ISSUE TITLE
tye build cannot work when source code is on another drive

### ISSUE THREAD

**Issue by bravecobra** [dotnet/tye#189]
### Describe the bug
I followed the sample but generated the code on my D: drive. The build command failed when tye tried to generate the Dockerfile for each project.

```
D:\tmp\microservice> tye build
Reading Project Details...
Reading Project Details...
Processing Service 'frontend'...
    Compiling Services...
    Publishing Project...
        Created Publish Output: 'D:\tmp\microservice\frontend\bin\Release\netcoreapp3.1\publish'
    Building Docker Image...
            unable to prepare context: unable to get relative Dockerfile path: Rel: can't make C:\Users\james\AppData\Lo
cal\Temp\tmp1413.tmp relative to D:\tmp\microservice\frontend\bin\Release\netcoreapp3.1\publish
Drats! 'build' failed:
        'docker build' failed.
```
### Further technical details
version 0.1.0-alpha.20168.8+f45b9444d4894009bde48e2a3411a52dd09497b1

**Comment by bravecobra** [comment]
Seems similar to #153

**Comment by rynowak** [comment]
This should be fixed as of `0.1.0-alpha.20174.3` - I don't have easy access to a windows machine to test it right now, so please let us know if you hit issues.
